version: "3.3"

services:
  traefik_service:
    image: traefik
    container_name: traefik_container
    command: --docker --logLevel=ERROR
    ports:
      - "80:80"
      - "443:443"
    networks:
      - kudaki-docker-net
    volumes:
      - ./acme.json:/acme.json
      - ./traefik.toml:/traefik.toml
      - /var/run/docker.sock:/var/run/docker.sock
    restart: on-failure

networks:
  kudaki-docker-net:
    external: true